### ISSUE TITLE
Ember Simple Auth support

### ISSUE THREAD

**Issue by ismyrnow** [knownasilya/just-auth#9]
Once we get to 1.0, I'd like to build an authenticator and authorizer for Ember Simple Auth.

**Comment by ismyrnow** [comment]
I wouldn't want this to be part of this project, but another one that is related to it. If we end up using the 1.0 of this in a project, we'll need the Ember Simple Auth piece anyhow, so might as well make an official implementation.

**Comment by knownasilya** [comment]
We could just link to it from the README as a related project.

**Comment by ismyrnow** [comment]
Agreed. Since that's something I will need before using 1.0, I think I will leave this issue open at least until we create the project.
